NickvisionTubeConverter.Shared/Models/Configuration.cs

```diff
@@ -49,7 +49,7 @@ public Configuration()
             Directory.CreateDirectory(ConfigDir);
         }
         Theme = Theme.System;
-        PreviousSaveFolder = "";
+        PreviousSaveFolder = Environment.GetFolderPath(Environment.SpecialFolder.MyVideos);
```

Comment:
I don't think it is the best choice to default to `MyVideos`. What is users use `Tube Converter` for downloading videos as audio files...`MyVideos` wouldn't make sense here. Best to leave it blank and have it set when they first chose the folder, no?

Reply:
This value is used only once - when a user opens add download dialog for the first time, then it gets replaced by whatever folder they choose after starting a download. And `Videos` seems to be sane default, because probably most users will want to download a video, not audio. Setting this to `home` or anything else "neutral" is even worse, because it will not be appropriate for most cases, I think that we might expect with high chances that users want to save files in eiher `Videos` or `Music` directory or somewhere on external drive, and the first one, while not being perfect for all use-cases, is the best default.
Setting it to blank doesn't solve anything but creates 2 problems:
1. All users will be forced to select a folder on the first download, so it doesn't help anybody, but `Videos` folder can be suitable for a lot of users;
2. Controller now expects `PreviousSaveFolder` to never be empty, because when it sees that URL has changed it generates new path with video title as filename. If there are no folder set, it can't generate a correct path.

Reply:
Ah since the controller expects a value now then no worries we can leave it as is :)